go.mod

```diff
@@ -3,41 +3,35 @@ module github.com/triggermesh/tm
 go 1.13
 
 require (
-	contrib.go.opencensus.io/exporter/ocagent v0.6.0 // indirect
-	contrib.go.opencensus.io/exporter/stackdriver v0.12.9 // indirect
 	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
-	github.com/dsnet/compress v0.0.1 // indirect
-	github.com/frankban/quicktest v1.7.2 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
+	github.com/json-iterator/go v1.1.10 // indirect
 	github.com/knative/build v0.7.0
 	github.com/knative/pkg v0.0.0-20190624141606-d82505e6c5b4
 	github.com/mholt/archiver v2.1.0+incompatible
-	github.com/nwaples/rardecode v1.0.0 // indirect
 	github.com/olekukonko/tablewriter v0.0.4
-	github.com/openzipkin/zipkin-go v0.2.2 // indirect
 	github.com/pierrec/lz4 v2.4.0+incompatible // indirect
-	github.com/robfig/cron v1.2.0 // indirect
-	github.com/sirupsen/logrus v1.4.2
+	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/afero v1.2.2
-	github.com/spf13/cobra v0.0.5
-	github.com/stretchr/testify v1.4.0
-	github.com/tektoncd/pipeline v0.11.2
+	github.com/spf13/cobra v1.0.0
+	github.com/stretchr/testify v1.5.1
+	github.com/tektoncd/pipeline v0.14.0
 	github.com/tektoncd/triggers v0.1.0
 	github.com/tidwall/gjson v1.3.2 // indirect
-	golang.org/x/crypto v0.0.0-20191206172530-e9b2fee46413
-	google.golang.org/genproto v0.0.0-20191028173616-919d9bdd9fe6 // indirect
+	golang.org/x/crypto v0.0.0-20200707235045-ab33eee955e0
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
+	golang.org/x/time v0.0.0-20200630173020-3af7569d3a1e // indirect
 	gopkg.in/src-d/go-git.v4 v4.13.1
-	gopkg.in/yaml.v2 v2.2.5
-	k8s.io/api v0.17.2
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.0
-	k8s.io/utils v0.0.0-20191030222137-2b95a09bc58d // indirect
-	knative.dev/eventing v0.12.0
-	knative.dev/eventing-contrib v0.11.2
-	knative.dev/pkg v0.0.0-20200207155214-fef852970f43
-	knative.dev/serving v0.12.0
-	sigs.k8s.io/controller-runtime v0.4.0 // indirect
+	gopkg.in/yaml.v2 v2.3.0
+	k8s.io/api v0.18.5
+	k8s.io/apimachinery v0.18.5
+	k8s.io/client-go v12.0.0+incompatible
+	k8s.io/klog/v2 v2.3.0 // indirect
+	k8s.io/utils v0.0.0-20200619165400-6e3d28b6ed19 // indirect
+	knative.dev/eventing v0.15.0
+	knative.dev/eventing-contrib v0.15.0
+	knative.dev/pkg v0.0.0-20200708171447-5358179e7499
+	knative.dev/serving v0.15.0
```

Comment:
I used v0.15.0 knative version since it brought less conflicts to this PR, which scope initially was not about deps update. I'll update dependencies to the latest versions as a part of a separate [PR](https://github.com/triggermesh/tm/pull/191)